### ISSUE TITLE
Question about reproducing the results on Assist09.

### ISSUE THREAD

**Issue by erving666** [UpstageAI/cl4kt#3]
I did not achieve the 0.7624 as reported in the paper after running the cloned code.
Here's my results:

> AUC ACC RMSE
> 0.73645  0.71828  0.43580

And my configs:

cl4kt_config:
>   hidden_size: 64
>   num_blocks: 2
>   num_attn_heads: 8
>   kq_same: True
>   final_fc_dim: 512
>   d_ff: 1024
>   l2: 0.0
>   dropout: 0.2
>   reg_cl: 0.1
>   mask_prob: 0.2
>   crop_prob: 0.3
>   permute_prob: 0.3
>   replace_prob: 0.3
>   negative_prob: 1.0
>   temp: 0.05
>   hard_negative_weight: 1.0

train_config:
>   log_wandb_fold: True
>   sequence_option: "recent"
>   seq_len: 100
>   batch_size: 128
>   eval_batch_size: 512
>   num_epochs: 300
>   print_epochs: 1
>   max_grad_norm: 2.0
>   learning_rate: 0.001
>   optimizer: adam

**Comment by aporia3517** [comment]
Hello. I appreciate your interest in my research. As I recall, there are two versions of the ASSIST09 dataset. The first is the non-deduplicated version used in the original Deep Knowledge Tracing paper. In contrast, I used the dataset after pre-processing, such as removing duplicates.
